Update yew-router examples, make them easier to run (#1401)
yewstack/yew: yew-router/examples/README.md

- switch - Various examples for how to construct routes with the router.

## Running

> There are two ways to run, with Rust servers or with Python.

Reviewer: ```suggestion
To run the examples you'll need to spin up a web server; two possible ways of doing this are either using a Rust webserver or the built-in Python webserver (we suggest this because Python is installed by default on most systems).
```

> ### Running with Rust (warp / actix)

Reviewer: These are proper nouns.
```suggestion
### Running with Rust (Warp / Actix)
```

> Details on how to build and run these examples can be found in the `README.md` under `servers/`.

Reviewer: ```suggestion
Details on how to build and run these examples can be found in the `README.md` contained in the `servers/` directory.
```

Using the router in its expected use case (not fragment routing) requires that the server respond to requests for
resources at URLs that are routes within the router with the index.html of the application.

> ### Running with Python

Reviewer: ```suggestion
### Serving the files using Python
```

> In applicable directories, launch the `run.sh` shell script.

Reviewer: ```suggestion
In applicable directories, run the `run.sh` shell script.
```
